### DISCUSSION TITLE
iOS Crash when stopped

### DISCUSSION THREAD

**Discussion by brianalsum** [advplyr/audiobookshelf#2716] [General]
I use ABS all day! everyday! i live in my audiobooks. However, I noticed that if the phone is locked (in my pocket) and I stop the book with my AirPods/Car remote I have to pull my phone out and re-open the app to start the book again... some times I can get away without having to go through all that, but that's maybe just 20% of the time. I didn't know if this was a Q&A or more of just a "current level of development" discussion

**Comment by gdesanto** [comment]
I've had this problem since the begining, Its starting to get real annoying actually.  Probably because its much more prevalent for me now. 

I'd say 80% of the time, I can't restart an audiobook without opening my phone, then the app, and hitting play.   

When i pull out the phone(iphone), it doesn't even display the play button on the lock screen, like the phone doesn't even know it was playing a book before. 

This happens to both downloaded local content and streamed books from the server.
